### ISSUE TITLE
[doc request] Kind was unable to start the cluster due to x509: certificate signed by unknown authority

### ISSUE THREAD

**Issue by rumstead** [kubernetes-sigs/kind#1702]
**What would you like to be documented**:
I had a similar issue to what was being described [here](https://github.com/kubernetes-sigs/kind/issues/1407).

My laptop uses a proxy and my kind was unable to start the control plane. I use the latest and greatest of kind, docker, and kubectl. I could see the `kindest/node` container running but when I did a `kubectl get pods -A`, no pods.

I read through the kind documentation, specifically around proxies. It mentioned adding docker to your `no_proxy`. However, this didn't solve my issue. After increasing my docker desktop resources and going through a couple of issues and was able to get better logs. 

```
$ kind create cluster --retain -v 1
$ kind export logs
```

From there, I was able to see the below error message, one I am far too familiar with. I then added `kind-control-plane` to my `no_proxy` and boom, it worked. 
```
Jul 02 15:08:29 kind-control-plane kubelet[289]: E0702 15:08:29.682481     289 reflector.go:178] k8s.io/kubernetes/pkg/kubelet/config/apiserver.go:46: Failed to list *v1.Pod: Get https://kind-control-plane:6443/api/v1/pods?fieldSelector=spec.nodeName%3Dkind-control-plane&limit=500&resourceVersion=0: x509: certificate signed by unknown authority
```

```
$ kind version                                                         
kind v0.8.1 go1.14.2 darwin/amd64

$ docker version                                                        
Client: Docker Engine - Community
 Azure integration  0.1.4
 Version:           19.03.12
 API version:       1.40
 Go version:        go1.13.10
 Git commit:        48a66213fe
 Built:             Mon Jun 22 15:41:33 2020
 OS/Arch:           darwin/amd64
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          19.03.12
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.10
  Git commit:       48a66213fe
  Built:            Mon Jun 22 15:49:27 2020
  OS/Arch:          linux/amd64
  Experimental:     true
 containerd:
  Version:          v1.2.13
  GitCommit:        7ad184331fa3e55e52b890ea95e65ba581ae3429
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683

$ kubectl version                                                  
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.5", GitCommit:"e6503f8d8f769ace2f338794c914a96fc335df0f", GitTreeState:"clean", BuildDate:"2020-06-27T00:38:11Z", GoVersion:"go1.14.4", Compiler:"gc", Platform:"darwin/amd64"}
```

**Why is this needed**:
Save others the trouble of having to diagnose this. 

If this is documented somewhere, it would be nice to put it under the `Known Issues`. I am happy to write something up and contribute it.

**Comment by rumstead** [comment]
TL;DR: If you are behind a proxy and see `x509: certificate signed by unknown authority` errors in your kubelet log, add `kind-control-plane` to your `no_proxy`.

**Comment by BenTheElder** [comment]
kind-control-plane is injected into no_proxy automatically in the next release
